app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1343,7 +1343,7 @@ public void openMainPage() {
 
   private void setUpWebView() {
     getCurrentWebView().getSettings().setJavaScriptEnabled(true);
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
I think you should remove the static import of `VERSION_CODES.KITKAT` and keep this line as it is. This will save us 1 line and it is more readable.